forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -616,15 +616,17 @@ public boolean changeCardState(final String mode, final String customState, fina
             // run valid Replacement here
             getGame().getReplacementHandler().run(ReplacementType.Transform, AbilityKey.mapFromAffected(this));
 
-            // do the Transform trigger here, it can also happen if the resulting state doesn't change
-            // Clear old dfc trigger from the trigger handler
-            getGame().getTriggerHandler().clearActiveTriggers(this, null);
-            getGame().getTriggerHandler().registerActiveTrigger(this, false);
+            if (!mode.equals("Convert")) {
```

Comment:
part of these changes seem old/wrong again?

Reply:
I had a weird rebase in there, should be fixed now